bin/omakub-sub/migrate.sh

```diff
@@ -1,5 +1,6 @@
 cd $OMAKUB_PATH
 last_updated_at=$(git log -1 --format=%cd --date=unix)
+git checkout stable
```

Comment:
One thing I'm not sure about is whether omakub upgrade is supposed to be usable if you've originally installed `master` rather than `stable`. If so, this line might be undesirable.

Maybe it's better to drop this line, and just rely on the fixed `boot.sh`...

Reply:
Actually ,yes, on second thought I think you're right.

Reply:
Fixed.

Reply:
Thanks! Apologies for not spotting the impact to the upgrade flow in the original PR. Probably warrants a mention in the release notes to instruct users to run `cd $OMAKUB_PATH && git checkout stable` if they encounter "You are not currently on a branch" errors when trying to upgrade.